sonar-flex-plugin/src/main/java/org/sonar/plugins/flex/cobertura/CoberturaReportPasrer.java

```diff
@@ -20,29 +20,28 @@
 package org.sonar.plugins.flex.cobertura;
 
 import com.google.common.collect.Maps;
+import static java.util.Locale.ENGLISH;
 import org.apache.commons.lang.StringUtils;
 import org.codehaus.staxmate.in.SMHierarchicCursor;
 import org.codehaus.staxmate.in.SMInputCursor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.sonar.api.batch.SensorContext;
+import org.sonar.api.batch.fs.FilePredicates;
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.measures.CoverageMeasuresBuilder;
 import org.sonar.api.measures.Measure;
-import org.sonar.api.resources.Project;
-import org.sonar.api.resources.Resource;
-import org.sonar.api.scan.filesystem.ModuleFileSystem;
+import static org.sonar.api.utils.ParsingUtils.parseNumber;
 import org.sonar.api.utils.StaxParser;
 import org.sonar.api.utils.XmlParserException;
+import org.sonar.plugins.flex.core.Flex;
 
-import javax.annotation.Nullable;
 import javax.xml.stream.XMLStreamException;
 import java.io.File;
 import java.text.ParseException;
 import java.util.Map;
 
-import static java.util.Locale.ENGLISH;
-import static org.sonar.api.utils.ParsingUtils.parseNumber;
-
 public class CoberturaReportPasrer {
```

Comment:
@vilchik-elena indeed thanks, fixed in 9b937d42451e8db944cd344c1ffa046fb9357086.